assets/core/0000_10_csr_approver_clusterrole.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Would you please extend the commit message clarifying on where all the new assets comes from and why they are needed?

Reply:
+1